Tests/DataPipeline/UnitTest.swift

```diff
@@ -0,0 +1,32 @@
+@testable import CioDataPipelines
+@testable import CioInternalCommon
+import Foundation
+@testable import Segment
+import SharedTests
+
+class UnitTest: SharedTests.UnitTest {
+    // When calling CustomerIOInstance functions in the test functions, use this `CustomerIO` instance.
+    // This is a workaround until this code base contains implementation tests. There have been bugs
```

Comment:
I don't quite understand this, and is it still needed?

Reply:
Not exactly how it is written. We can probably use `CustomerIO.shared` too but let's stick to it for unit tests. Improved the comment.